src/utils/deployTFhubValidator.ts

```diff
@@ -0,0 +1,138 @@
+import type TFhubValidator from "../types/TFhubValidator";
+import { v4 } from "uuid";
+import type { IProfile } from "../types/Profile";
+import { Network } from "../types/kubernetes";
+
+import createNetwork from "./createNetwork";
+import deploy from "./deploy";
+import rootFs from "./rootFs";
+import checkVMExist from "./prepareDeployment";
+
+function getNetwork() :string {
+  const networks = ['dev', 'qa', 'test', 'main'];
+  const host = window.location.host;
+  let netWork = '';
+  networks.includes(
+    host.split('.')[1])
+    ? netWork = host.split('.')[1]
+    : netWork = 'main';
+  return netWork;
+}
+
+function defaultEnvVars(host: string){
+  // Replace dev with main when you deploying the validator on localhost. 
+  const env = {
+    dev: {
+      chainId: "threefold-hub-testnet",
+      gravityAddress: "0x7968da29488c498535352b809c158cde2e42497a",
+      ethereumRpc: "https://data-seed-prebsc-2-s1.binance.org:8545",
+      persistentPeers: "67bd27ada60adce769441d552b420466c2082ecc@185.206.122.141:26656",
+      genesisUrl: "https://gist.githubusercontent.com/OmarElawady/de4b18f77835a86581e5824ca954d646/raw/8b5052408fcd0c7deab06bd4b4b9d0236b5b1e6c/genesis.json",
+    },
+    qa: {
+      chainId: "",
+      gravityAddress: "",
+      ethereumRpc: "",
+      persistentPeers: "",
+      genesisUrl: "",
+    },
```

Comment:
I think we should set the config of QA the same as dev, @xmonader what do you think?